paternoster/types/__init__.py

```diff
@@ -15,6 +15,9 @@ def __call__(self, val):
 
         if self._wildcard and val.startswith('*.'):
             val = val[2:]
+            
+        if val.endswith('.'):
+            val = val[:-1]
```

Comment:
This leaves `domain` with the dotted `example.com.` value. Since there is no real difference between `example.com` and `example.com.`, I think we should strip the dot and thus normalize given domain names. Please change this so the value of `domain` is updated as well.

Reply:
I'm guessing we don't want to strip the wildcard?

Reply:
yep, because the playbook needs to know that a wildcard was passed. Otherwise that information is lost. We could strip the wildcard, but change the return value to something like `{"wildcard": true, "domain": "google.com"}`, but at the moment it works by just leaving it in and ignoring it during validation.